person.rb

```diff
@@ -1,6 +1,10 @@
 # create a person class 
+require_relative 'Nameable'
+require_relative 'CapitalizeDecorator'
+require_relative 'TrimmerDecorator'
+require_relative 'Decorator'
 
-class Person
+class Person < Nameable
```

Comment:
- [ ] It would be better if we are able to test the code snippet below, however, there are some syntax errors we need to address. Here is what I did, I created a `main.rb` file and required all the classes and added the code as seen in the screenshot
![Screenshot from 2022-08-31 12-56-00](https://user-images.githubusercontent.com/46388113/187687504-1d80c937-6118-433b-ac38-86989413fe1c.png)